src/pages/gallery/gallery.tsx

```diff
@@ -29,12 +29,12 @@ export default function Gallery({ games, tags }: { games: GameMetadata[], tags:
 
 	function filterTags(games: GameMetadata[]): void {
 		if (tagFilter === "") {
-			let otherGames = [...games];
+			let otherGames = [...games].map(game => ({ ...game, show: true }));
 			sortGames(otherGames, sortOrder as SortOrder);
 			setGamesState(otherGames as GalleryGameMetadata[]);
 			return;
 		}
-		setGamesState(games.filter(game => game.tags.includes(tagFilter)) as GalleryGameMetadata[]);
+		setGamesState(games.filter(game => game.tags.includes(tagFilter)).map(game => ({ ...game, show: true })) as GalleryGameMetadata[]);
```

Comment:
Can this be expressed without the elipsis syntax?  I do not this the conciseness here outweighs readability.